cosima_cookbook/explore.py

```diff
@@ -775,14 +798,24 @@ def _load_experiment(self, b):
         Open an Experiment Explorer UI with selected experiment
         """
         if self.expt_selector.value is not None:
-            self.ee = ExperimentExplorer(session=self.session,
+            self.ee = ExperimentExplorer(session=self.session, 
                                          experiment=self.expt_selector.value)
             self.expt_explorer.children = [self.ee]
 
+    def data(self):
+        """
+        Return xarray DataArray if one has been loaded in ExperimentExplorer 
+        """
+        if self.ee is None:
+            print('Cannot return data if no experiment has been loaded')
+            return None
+
+        return self.ee.data()
+
 class ExperimentExplorer(VBox):
 
     session = None
-    data = None
+    loaded_data = None
```

Comment:
Maybe private member `_loaded_data` if we only want to expose through the `data` method/property?

Reply:
Probably. Will change.